bigtop_toolchain/files/grpc-java-1.28.0-add-support-for-ppc64le.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/compiler/build.gradle b/compiler/build.gradle
+index 60d3a43..55acee0 100644
+--- a/compiler/build.gradle
++++ b/compiler/build.gradle
+@@ -105,7 +105,6 @@ model {
+                     // Link other (system) libraries dynamically.
+                     // Clang under OSX doesn't support these options.
+                     linker.args "-Wl,-Bstatic", "-lprotoc", "-lprotobuf", "-static-libgcc",
+-                            "-static-libstdc++",
```

Comment:
Instead of removing this, adding libstdc++-static to the packages to be installed seemed to resolve the compiling issue on CentOS 7.
https://github.com/sekikn/bigtop/commit/b94539c280df2b698a9d55db0cca753634d2d1b1#diff-9337fb34935804ab7b56cd89ae94e14491ad4c64804daa9b1843eeec336772edR67

Reply:
Not sure that would work on Ubuntu. But the fix is a backport of an upstream patch https://github.com/grpc/grpc-java/pull/8141